### PR TITLE
Group brackets

### PR DESCRIPTION
**Pull request by markschwindt**
This PR introduces brackets for cast list groups as requested in #68.

### CODE REVIEW

**Comment by lehkost** [comment]
Looking good to me! 💯

erster-versuch-group-brackets-example

File: tailwind.config.js [hunk 1]
```diff
@@ -44,6 +44,9 @@ module.exports = {
           },
         },
       },  
+      height: {
+        "calc-full-0.75": "calc(100% - 0.75rem)",
```
Comment (cmil) [inline tailwind.config.js:48]: Lets use single quotes for strings in all Javascript code. The exception would be properties or attributes in JSX (`className="foo"`).

**Review by cmil — CHANGES_REQUESTED** [review]
Very cool! Looks good to me too.

For just one tiny detail see comment below.

File: tailwind.config.js (hunk 1 above)
Comment (markschwindt) [reply tailwind.config.js:48]: Sure!

**Review by cmil — APPROVED** [review]

**Pull request merged**